### ISSUE TITLE
Install target does not copy DLLs on windows

### ISSUE THREAD

**Issue by jcar87** [iamantony/qtcsv#47]
When building as shared libraries on windows (the default), the "install" target doesnt copy the DLLs (typically to a "bin" directory).

This is because in the install command  (CMakeLits.txt) there is no instructions to copy "runtime" files.

It should look something like this:
```
install(TARGETS ${LIBRARY_NAME} EXPORT ${LIBRARY_NAME}Config
        LIBRARY DESTINATION lib
        RUNTIME DESTINATION bin
        ARCHIVE DESTINATION lib)

```

**Comment by iamantony** [comment]
Hello @jcar87!

Sorry for a late reply. Thank you for the issue! Fixed in commit 5ba4b2c5613d6a0ef6f5832edbe9145c3e092b2f.

Unfortunately, `make install` command will not work with qmake build on Windows. Does anyone know how to fix it?

**Comment by siakc** [comment]
This is not true. I have recently installed "requests" with that command.
